src/runtime/renderscript_dev.ll

```diff
@@ -0,0 +1,29 @@
+target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
```

Comment:
Probably should add that explanation as a comment in the code.

On Tue, Apr 28, 2015 at 8:31 PM, Alexander Aprelev <notifications@github.com
> wrote:

> In src/runtime/renderscript_dev.ll
> <https://github.com/halide/Halide/pull/761#discussion_r29308276>:
>
> > @@ -0,0 +1,29 @@
> > +target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
>
> Actually, those directives are needed in renderscript_dev.ll file. I put
> them back in 09e6be0
> <https://github.com/halide/Halide/commit/09e6be0bc128231a6f719499c983a58c939b8aaf>
> .
>
> Reason is that Renderscript code generation is done via compile_to_src, on
> a different path from CodeGen_LLVM::compile(), so
> CodeGen_Renderscript_Dev::get_target_triple() and get_data_layout() are not
> really being used. The methods have to be defined, though, as they are
> declared as abstract in parent classes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/761/files#r29308276>.
>